plugins/modules/cloud/misc/proxmox.py

```diff
@@ -115,6 +115,12 @@
     description:
       - specifies network interfaces for the container. As a hash/dictionary defining interfaces.
     type: dict
+  features:
+    description:
+      - Specifies a list of features to be enabled. For valid options, see U(https://pve.proxmox.com/wiki/Linux_Container#pct_options).
```

Comment:
As discussed in https://github.com/ansible-collections/community.general/issues/816#issuecomment-725439886 it would be good to make some notes about features which only work on privileged or unprivileged containers.
```suggestion
      - Specifies a list of features to be enabled. For valid options, see U(https://pve.proxmox.com/wiki/Linux_Container#pct_options).
      - Some features can only be turned on privileged or unprivileged containers.
```
I'm not sure about formatting of my sentence, please change it whatever you find suitable.